test/tests/api/v2_0/obm_settings.py

```diff
@@ -0,0 +1,101 @@
+from config.api2_0_config import *
+from on_http_api2_0 import ApiApi as Api
+from on_http_api2_0 import rest
+from modules.logger import Log
+from json import dumps, loads
+
+LOG = Log(__name__)
+
+"""
+Class to abstract the RackHD Out-of-Band settings 
+"""
+class obmSettings(object):
```

Comment:
Hrm, API method name differences.  Still might be worth looking into making this module common.